deployments/sub-bridge-types.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Let's just have a single `json` file with all types please. If it grows to big we can later have it auto-crated by CI from multiple smaller ones, but for now, for simplicity I'd rather have just one.

Reply:
I had to rename a type in the Ethereum pallet for this, but since Rialto has both Ethereum and Substrate bridge pallets (and will thus need both types) I think it's okay